rdr_service/genomic/genomic_job_components.py

```diff
@@ -548,6 +550,13 @@ def _ingest_aw4_manifest(self, rows):
 
                 self.member_dao.update(member)
 
+                dispatch_task(
+                    endpoint='update_enrollment_status',
+                    payload={
+                        'participant_id': member.participantId
+                    }
+                )
```

Comment:
Sorry, this works but would you mind changing this to the one that is attached to the controller for consistency in the genomics system. You will just need to pass the queue that this is defaulting to. 

`self.controller.execute_cloud_task`

Reply:
No worries, thanks for letting me know about the method. Updated!

Reply:
Hey last thing, this method defaults to the `genomics` queue. You will need to update this to be in the `resource-tasks` queue if you want it there, but I dont think the queue really matter for which endpoints can be accessed from it.

Reply:
Definitely meant to have that go to another queue. It seems cleaner to leave the genomics queue as genomics tasks. Thanks for catching that.